tests/system/provisioning/vars/credentials.yml

```diff
@@ -1,16 +1,5 @@
 ### Ansible connection vars
-ssh_username: wazuh
-ssh_password: T3stP4ssw0rd--
-key_pair_name: devops-dev
-
-ssh_key_path_private: ~/.ssh/devops-dev.pem
-ssh_key_path_public: ~/.ssh/devops-dev.pub
-
-custom_ssh_key: ""
+ssh_username: jenkins
+key_pair_name: Jenkins
 
 windows_user: Administrator
-windows_password: ""
-
-hosts_classified_destination: "./ec2_demo_hosts_classified"
```

Comment:
These destinations define the path destination for the provisioning playbooks. Please detail motivation for removing them.

Reply:
Referred to https://github.com/wazuh/wazuh-qa/pull/509#discussion_r382036642